_shows/20_21/speaking_solo.md

```diff
@@ -0,0 +1,43 @@
+---
+# Check out the Show Docs for more information
+# -- https://history.newtheatre.org.uk/docs/show/
+# Remove or comment any lines that are not being used
+
+title: NNT Presents 'Speaking Solo'
+playwright: Alexander Hogg, Emi Thackray, Geethija Ayiliath, Sunenna Sohal and Ellie Harding
+student_written: true
+period: Autumn
+season: Online
+season_sort: 450
+date_start: 2021-03-17
+date_end: 2021-03-17
+
+cast:
+- role: Actor 'Red'
+  name: Megan Smith
+- role: Actor 'The Universe Will Take Care of You' 
+  name: Ace Edwards
+- role: Actor 'A Momentary Spark Between Neurons'
+  name: jakmes Eykyn
+- role: Actor 'Nighthawk' 
+  name: Jack Titley
+
+crew: # Omit certain roles as required
```

Comment:
The writers can then go in here with a note for which monologue they did

Reply:
done!